player/player.tscn

```diff
@@ -9,11 +9,19 @@ size = Vector2(128, 128)
 [node name="Player" type="CharacterBody2D" groups=["Player"]]
 z_index = 1
 collision_layer = 2
+collision_mask = 41
```

Comment:
What is the need for changing the collision_mask here?

Reply:
The player previously could not collide with holes in the ground. Since the player interacting with holes is relevant to the dodge roll mechanic, this has been changed.